migrations/2021-11-23-153753_add_invite_only_columns/up.sql

```diff
@@ -0,0 +1,19 @@
+-- Add columns to site table
+alter table site add column require_application boolean not null default false;
+alter table site add column require_email boolean not null default false;
+alter table site add column application_question text;
+
+-- Add pending to local_user
+alter table local_user add column accepted_application boolean not null default false;
+alter table local_user add column verified_email boolean not null default false;
+
+create table registration_application (
+  id serial primary key,
+  local_user_id int references local_user on update cascade on delete cascade not null,
+  answer text not null,
+  acceptor_id int references person on update cascade on delete cascade,
+  accepted boolean not null default false,
+  deny_reason text,
+  published timestamp not null default now(),
+  unique(local_user_id)
+);
```

Comment:
@Nutomic take a look at these, I changed some of the column names to make more sense. If they look good I can continue on with it.

Reply:
`published` is the creation time? That one is a bit unclear, others are fine. Not sure if acceptor_id and, accepted and deny_reason are really necessary. The first doesnt really matter, instead of accepted I would use local_user.activated (which i'm adding right now for email verification), and deny_reason can just be sent by email without storing. Well maybe that one makes sense for other admins to view.